packages/manager/apps/web/client/app/private-database/translations/Messages_fr_FR.json

```diff
@@ -417,65 +417,6 @@
   "privateDatabase_metrics_time_range_DAY": "Dernières 24 heures",
   "privateDatabase_metrics_time_range_WEEK": "Dernière semaine",
   "privateDatabase_metrics_time_range_MONTH": "Dernier mois",
-  "privateDatabase_order_sql_main_title": "Commander une base de données SQL",
```

Comment:
We could clean the translations targets files when merged.

Reply:
What do you mean ? Remove them from this PR and do it later in another PR on `develop` ? I considered it and didn't know what was the best procedure

Reply:
We will automatize this process. It's related to #2284.

For now we can keep this as is.